app/models/manageiq/providers/hawkular/middleware_manager/event_catcher/runner.rb

```diff
@@ -1,20 +1,21 @@
 class ManageIQ::Providers::Hawkular::MiddlewareManager::EventCatcher::Runner <
   ManageIQ::Providers::BaseManager::EventCatcher::Runner
 
+  TAG_EVENT_TYPE    = "miq.event_type".freeze # required by fetch
+  TAG_RESOURCE_TYPE = "miq.resource_type".freeze # optionally provided when linking to a resource
+
   def initialize(cfg = {})
     super
 
-    # Accept events with an event category in the keyset and whose event message matches the (optional) pattern value
-    # - key = event category
-    # - value (optional) event message regex
-    @whitelist = {
-      'Hawkular Deployment' => nil
-    }.freeze
-
-    # Some ems events will link to objects in miq inventory. This maps the event category to the object type
-    @types = {
-      'Hawkular Deployment' => MiddlewareServer.name
-    }.freeze
+    # Supported event_types (see settings.yml)
+    @whitelist = [
+      # summary
+      'hawkular_deployment.error',
+      'hawkular_event.critical', # general purpose critical/summary level event
+      # detail
+      'hawkular_deployment.ok',
+      'hawkular_event' # # general purpose deltail level event
+    ].freeze
```

Comment:
a ruby hint: as you are only using this array for `include?` you can use a `Set`
```ruby
@whitelist = [1,2].to_set.freeze 
```